receiver/apachereceiver/factory.go

```diff
@@ -32,9 +32,7 @@ func NewFactory() receiver.Factory {
 
 func createDefaultConfig() component.Config {
 	return &Config{
-		ScraperControllerSettings: scraperhelper.ScraperControllerSettings{
-			CollectionInterval: 10 * time.Second,
```

Comment:
Looks like this was missed and effectively the Apache receiver's default collection interval has changed from 10s to 60s?

Reply:
Shit, I can revert in a follow up PR.